ush/forecast_postdet.sh

```diff
@@ -180,6 +195,16 @@ EOF
 	    O3FORC=global_o3prdlos.f77
 	fi
 	H2OFORC=${H2OFORC:-"global_h2o_pltc.f77"}
+        ####
+        # copy CCN_ACTIVATE.BIN for Thompson microphysics
+        if [ $imp_physics -eq 8 ]; then
+        $NCP $FV3INP/CCN_ACTIVATE.BIN  CCN_ACTIVATE.BIN
+        ####
+        $NCP /scratch1/NCEPDEV/stmp2/Lin.Gan/RUNDIRS/gsdsuite-NEW/2019100900/gfs/fcst.383441/freezeH2O.dat  freezeH2O.dat
```

Comment:
What this section of code is for?  I thought Lin was trying to remove things from being copied from personal directories? This seems like it's adding that in here?

Reply:
One of the CCPP suite "FV3_GSD_V0" needs additional files from outside. I agree that this shouldn't be the way forward. I will contact Lin asking to put this file into a permanent place.

Reply:
These files are now part of FIX_AM